requirements.txt

```diff
@@ -1,3 +1,4 @@
 ###### Requirements without Version Specifiers ######
-transitfeed
-pandas
\ No newline at end of file
+partridge
+pytest
+functools32
```

Comment:
will fail on python 3.6; see https://www.python.org/dev/peps/pep-0508/#environment-markers

Reply:
Added a python version specifier for the requirements.txt.

Reply:
I think the Zip File issue needs to be handled on the Partridge side of the fence. Let me get a clean PR into @invisiblefunnel and see if we can resolve the issue there. 

There is no clean spot to insert it on the Fast Trips side and manage the pointer and clean-up at the end. It could be done on this side, but seems far easier on the Partridge side of the fence.